### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by Anchor-ZhuoHU** [OwO-Network/DLX#92] [Q&A] [answered]
Hi, and thanks for your great app. However, I want to uninstall it now and don't want it listening to my port 1188 anymore. I went through a bit in the source code but didn't figure out where to rm the relative files. Could you please update such a method?

**Comment by missuo** [comment]
May I ask what method you used for installation, and on which platform did you install it?

**Reply by Anchor-ZhuoHU** [reply]
Thanks for your quick reply. I installed it on macOS Sonoma 14.2.1. I remembered I used Homebrew to install but I'm not quite clear. However, I do uninstall from homebrew using `brew uninstall deeplx` and it prompted uninstall correctly. However, when I directly tried to GET the URL which used to listen to, it still listens and I don't know why. I don't want to kill manually every time I restart my computer. There should be some service remaining in my machine that haven't been deleted.

<img width="1205" alt="Screenshot 2024-01-23 at 10 43 59 PM" src="https://github.com/OwO-Network/DeepLX/assets/138105626/6b6294e5-0e8e-4e1d-8edc-8ac3f87cb11b">

**Reply by missuo** [reply, marked as answer]
```bash
brew services stop deeplx
brew uninstall deeplx
```

try this command.
